code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -442,19 +450,19 @@
 					wounding_dmg *= (easy_dismember ? 1 : 0.75)
 				if((mangled_state & BODYPART_MANGLED_BONE) && try_dismember(wounding_type, wounding_dmg, wound_bonus, bare_wound_bonus))
 					return
-			// note that there's no handling for BIO_JUST_FLESH since we don't have any that are that right now (slimepeople maybe someday)
+			// note that there's no handling for BIO_FLESH since we don't have any that are that right now (slimepeople maybe someday)
 			// standard humanoids
 			if(BIO_FLESH_BONE)
 				// if we've already mangled the skin (critical slash or piercing wound), then the bone is exposed, and we can damage it with sharp weapons at a reduced rate
 				// So a big sharp weapon is still all you need to destroy a limb
-				if(mangled_state == BODYPART_MANGLED_FLESH && sharpness)
+				if((mangled_state & BODYPART_MANGLED_FLESH) && !(mangled_state & BODYPART_MANGLED_BONE) && sharpness)
 					playsound(src, "sound/effects/wounds/crackandbleed.ogg", 100)
 					if(wounding_type == WOUND_SLASH && !easy_dismember)
 						wounding_dmg *= 0.6 // edged weapons pass along 60% of their wounding damage to the bone since the power is spread out over a larger area
 					if(wounding_type == WOUND_PIERCE && !easy_dismember)
 						wounding_dmg *= 0.75 // piercing weapons pass along 75% of their wounding damage to the bone since it's more concentrated
 					wounding_type = WOUND_BLUNT
-				else if(mangled_state == BODYPART_MANGLED_BOTH && try_dismember(wounding_type, wounding_dmg, wound_bonus, bare_wound_bonus))
+				else if((mangled_state & BODYPART_MANGLED_FLESH) && (mangled_state & BODYPART_MANGLED_BONE) && try_dismember(wounding_type, wounding_dmg, wound_bonus, bare_wound_bonus))
```

Comment:
Why not make a BODYPART_MANGLED_BOTH?

Reply:
I am not fond of using == for bitfields, for the rare case where a variable has somehow been set to ALL or something silly like that. This is why I corrected every direct equality check I was able to reasonably get rid of.